src/main/java/org/hyperskill/hstest/dev/stage/BaseStageTest.java

```diff
@@ -194,7 +195,7 @@ public void start() {
                     assertTrue(errorMessage, result.isCorrect());
                 }
             }
-        } catch (Exception ex) {
+        } catch (Exception | Error ex) {
```

Comment:
Seems like there should be just `Throwable`

Reply:
Done.